C++/C.sublime-syntax

```diff
@@ -838,25 +841,28 @@ contexts:
               pop: true
             - include: preprocessor-disabled
 
+  preprocessor-if-true:
```

Comment:
While createing named contexts is generally a good thing, it appears unrelated with the topic if this PR.